pkgs/development/python-modules/onnx/default.nix

```diff
@@ -44,14 +44,13 @@ buildPythonPackage rec {
 
   buildInputs = [
     abseil-cpp
-    protobuf
+    python-protobuf
```

Comment:
The matching protobuf version should be taken from passthru https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/protobuf/4.nix#L119

Reply:
Ah thanks! How does that look?